### PR TITLE
Add a way to covert an array of nibbles to a byte array

### PR DESCRIPTION
**Pull request by diegomrsantos**

### CODE REVIEW

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java [hunk 1]
```diff
@@ -34,6 +35,33 @@ public static String toHexString(byte[] byteBuffer) {
...
 		}
 	}
+
+	/**
+	 * Convert an array of nibbles to a byte array. Starting from the beginning of the array, it converts each pair
+	 * of nibbles into a byte. The first nibble is the 4 most significant bits and the second one the 4 least.
+	 * <br/>
+	 * Examples: <br/>
+	 *	[0, 1, 0, 2] becomes [1, 2] <br/>
+	 *  [1, 1, 2, 2] becoenes [17, 34] <br/>
```
Comment (michalzajda) [inline radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java:45]: typo

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java [hunk 2]
```diff
@@ -34,6 +35,33 @@ public static String toHexString(byte[] byteBuffer) {
...
+	 *
+	 * @param nibbles nibbles array to be converted
```
Comment (michalzajda) [inline radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java:47]: repetition

**Review by michalzajda — APPROVED** [review]

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java (hunk 1 above)
Comment (diegomrsantos) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java:45]: fixed

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java (hunk 2 above)
Comment (diegomrsantos) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java:47]: not necessarily a repetition as the first one is the name of the parameter and second is a description, but I removed it anyway

**Pull request merged**